.circleci/config.yml

```diff
@@ -156,8 +148,7 @@ workflows:
             tags:
               only: /.*/
           requires:
-            - pip_install_36_browser
-            - pipenv_install_36_browser
+            - cli
```

Comment:
:+1:

Reply:
I just noticed that `test_build` requires the other demos, should I make it require `web_examples` as well?

Reply:
Depends if you want `test_build` to require `web_examples` to pass before continuing.  Doing so will prevent publication from continuing to PyPI on failure, for example.

I think it's a good idea to require these to pass as a form of acceptance testing, but I defer to your judgement on this, one.

Reply:
Agreed. I think this would be a good thing since we are essentially using the code similar to how an app would.